core/dbt/exceptions.py

```diff
@@ -849,6 +849,15 @@ def raise_patch_targets_not_found(patches):
     )
 
 
+# todo: fix error message to be useful & add test
+def raise_no_unique_id(patch):
```

Comment:
If we're not calling this anywhere, I think we can avoid adding it / causing future confusion